### ISSUE TITLE
Color contrast issue in FeedValidator should be a warning, not an error.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [SEBDYA/googletransitdatafeed#46]
```
Also, the text of the issue description should be properly spaced so that
it doesn't run together when displayed in HTML.
```

Original issue reported on code.google.com by `joe.hugh...@gmail.com` on 19 Mar 2008 at 11:51

**Comment by GoogleCodeExporter** [comment]
```
Fixed as of r648
```

Original comment by `joe.hugh...@gmail.com` on 27 Mar 2008 at 4:38
- Changed state: **Fixed**
